index.html

```diff
@@ -2276,22 +2284,133 @@ <h3 id="applying">
       </section>
       <section>
         <h3>
+          <code>purpose</code> member
+        </h3>
+        <p>
+          The <dfn><code>purpose member</code></dfn> is an <a>unordered set of
+          unique space-separated tokens</a> that are <a>ASCII
+          case-insensitive</a>. The allowed values the <a>icon purposes</a>.
+        </p>
+        <p>
+          When an <a>image object</a> is used as an <a>icon</a>, a developer
+          can specify that the image is intended to serve some special
+          <dfn>purpose</dfn> in the context of the host OS (i.e., for better
+          integration).
+        </p>
+        <p>
+          For example, as a <a data-lt="badge purpose">badge</a> or "pinned"
+          icon that is visually distinct, in color or form, from an
+          application's launch icon. The user agent uses the value of the
+          <a><code>purpose</code> member</a> as a hint to determine where and
+          how an <a>image object</a> is displayed. Unless declared otherwise by
+          the developer, a user agent can use an icon for <a>any purpose</a>.
+        </p>
+        <p>
+          The <dfn>icon purposes</dfn> are as follows:
+        </p>
+        <dl>
+          <dt>
+            <dfn data-lt="badge purpose">badge</dfn>:
+          </dt>
+          <dd>
+            A user agent can present this icon where space constraints and/or
+            color requirements differ from those of the application icon.
```

Comment:
Badge icons are intended to be applied on top of other image assets such as profile photos, in such cases where these might be more important than the app icon, e.g. when receiving a notification.